crates/wash-cli/src/app/mod.rs

```diff
@@ -341,3 +370,12 @@ fn show_model_history(results: VersionResponse) -> CommandOutput {
     map.insert("revisions".to_string(), json!(results));
     CommandOutput::new(output::list_revisions_table(results.versions), map)
 }
+
+fn show_model_status(model_name: String, results: StatusResponse) -> CommandOutput {
+    let mut map = HashMap::new();
+    map.insert("status".to_string(), json!(results));
```

Comment:
NIT: you could probably get away with `HashMap::from(...)` instead of making this temp var

Reply:
Definitely could, but we do it this way alllll over the place in wash, and I think for good reason. The `map.insert` calls make it easier to visually see each entry we put in the map. So at the very least for consistency, I'm going to leave this as-is